### PR TITLE
Add method to dynamically change rulesets and have changes reflect on map

### PR DESCRIPTION
**Pull request by jzaturensky**

### CODE REVIEW

**Comment by f2m2rd** [comment]
@jzaturensky where is the Jira Ticket for this?

**Comment by jzaturensky** [comment]
Created here - https://airmap.atlassian.net/browse/PLAT-2360

File: src/index.js [hunk 1]
```diff
@@ -179,6 +179,57 @@ export default class ContextualAirspacePlugin {
         return uniqBy(jurisdictions, 'id');
     }
 
+    /**
+    * Method that takes a list of preferred rulesets, queries the map for new
+    * jurisdictions and updates the state of the class with the state of jurisdictions
+    * and default selected rulesets. Additionally, map layers are updated here.
+    * @private
```
Comment (Skoryts) [inline src/index.js]: It would be nice, if you describe params and return value

File: src/index.js [hunk 2]
```diff
@@ -179,6 +179,57 @@ export default class ContextualAirspacePlugin {
...
+    */
+    updateRulesets(preferredRulesets, overrideRulesets, enableRecommendedRulesets) {
+        if (this.map.isMoving()) return
+        let jurisdictions = this.getJurisdictionsFromMap()
```
Comment (Skoryts) [inline src/index.js]: As I see, 'jurisdictions' is not changing through the method. It should be created as constant.

File: src/index.js [hunk 3]
```diff
@@ -179,6 +179,57 @@ export default class ContextualAirspacePlugin {
...
+                so we'll add the preferredRulesets to the map.
+            */
+            defaultSelectedRulesets.forEach(ruleset => this.addRuleset(ruleset));
+        } else {
+            /*
+                If selectedRulesets is not empty, we find the differences (to remove) and remove those.
+                We can send the rest of the rulesets to be added as we'll be checking if ruleset source already
+                exists before adding it.
+            */
+            let rulesetsToRemove = differenceBy(this.selectedRulesets, defaultSelectedRulesets, 'id');
```
Comment (Skoryts) [inline src/index.js]: You should use 'const' instead of 'let' if your value will not change.

**Pull request merged**